app/src/main/java/com/lagradost/cloudstream3/network/CloudflareKiller.kt

```diff
@@ -48,19 +50,26 @@ class CloudflareKiller : Interceptor {
 
     override fun intercept(chain: Interceptor.Chain): Response = runBlocking {
         val request = chain.request()
+        val response = chain.proceed(request)
```

Comment:
Suppose cookies are not null then this line always tries to do a request without cookies and only after that it shoots another request with cookies at line 67. This wastes a request.